grc/rds_parser.block.yml

```diff
@@ -42,6 +42,6 @@ templates:
     imports: import rds
     make: rds.parser(${log}, ${debug}, ${pty_locale})
     callbacks:
-    - reset(${reset})
+    - 'reset() # ${reset}'
```

Comment:
It's necessary to include `${reset}` here; otherwise GRC will never invoke the callback. I have made it a comment so that it is not passed into `reset()`. Here is the generated code:

```python
self.rds_parser_0.reset() # self.freq
```